package.json

```diff
@@ -32,7 +32,8 @@
     "eslint": "^8.25.0",
     "semantic-release": "^19.0.5",
     "typescript": "^4.8.4",
-    "vite": "3.1.8"
+    "vite": "3.1.8",
+    "vite-plugin-logseq": "^1.1.2"
```

Comment:
i think this not needed )

Reply:
It seems to work for me to generate dist/index.html which source localhost's server tor translate TS. I wonder what is the alternative way to test the code without npm run build every time code changes?

Reply:
i do every time `npm run build` and copy `dist` to plugins folder ((